test/main.js

```diff
@@ -179,6 +179,22 @@ test('ky.extend()', async t => {
 	await server.close();
 });
 
+test('ky.extend() throws when given non-object argument', t => {
+	[
+		true,
+		666,
+		'hello',
+		[],
+		null,
+		() => {},
+		Symbol('ky')
+	].forEach(value => {
+		t.throws(() => {
+			ky.extend(value);
+		});
```

Comment:
You should also assert that it throws the correct message and the type of the error. Use the object form of the `t.throws` assertions: https://github.com/avajs/ava#throwsasyncthrower-expected-message